dbt/scripts/export_models.py

```diff
@@ -92,9 +92,9 @@ def parse_args():
         log_group_name=args.log_to_cloudwatch_group,
     )
 
-    logger.info("Starting model export")
-
     try:
+        logger.info("Starting model export")
```

Comment:
I figure we might as well try to include as much as possible in the `try`/`except` block so we can catch and log as many errors as we can.